templates/addons/ddb/cf.yml

```
@@ -1,23 +1,73 @@
-{{.TableResourceName}}:
-  Type: AWS::DynamoDB::Table
-  DeletionPolicy: Retain
-  Properties:
-    TableName: !Sub ${App}-${Env}-${Name}-{{.TableName}}
-    AttributeDefinitions:
-      - AttributeName: {{.Name}}
-        AttributeType: "{{.DataType}}"{{end}}
-    BillingMode: PAY_PER_REQUEST
-    KeySchema: 
-      - AttributeName: {{.PartitionKey}}
-        KeyType: HASH{{ if .SortKey }}
-      - AttributeName: {{.SortKey}}
-        KeyType: RANGE{{end}}{{if .HasLSI}}
-    LocalSecondaryIndexes:{{range .LSIs}}
-      - IndexName: {{.Name}}
-        KeySchema:
-          - AttributeName: {{.PartitionKey}}
-            KeyType: HASH
-          - AttributeName: {{.SortKey}}
-            KeyType: RANGE
-        Projection:
-          ProjectionType: ALL{{end}}{{end}}
\ No newline at end of file
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+Resources:
+  {{.TableResourceName}}:
+    Type: AWS::DynamoDB::Table
+    DeletionPolicy: Retain
+    Properties:
+      TableName: !Sub ${App}-${Env}-${Name}-{{.TableName}}
+      AttributeDefinitions:
```

```suggestion
      AttributeDefinitions:{{range .Attributes}}
```

Just realized I missed this in the previous template PR.

---

a stupid question: should this be `{{range $attr := .Attributes}} and refer `$attr` below?